src/test/java/org/folio/rest/impl/protection/ProtectedEntityTestBase.java

```diff
@@ -0,0 +1,62 @@
+package org.folio.rest.impl.protection;
+
+import static org.folio.invoices.utils.ResourcePathResolver.INVOICES;
+import static org.folio.invoices.utils.ResourcePathResolver.INVOICE_LINES;
+import static org.folio.rest.RestVerticle.OKAPI_USERID_HEADER;
+import static org.folio.rest.impl.MockServer.addMockEntry;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.nullValue;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.folio.rest.impl.ApiTestBase;
+import org.folio.rest.impl.MockServer;
+import org.folio.rest.jaxrs.model.Invoice;
+import org.folio.rest.jaxrs.model.InvoiceLine;
+import org.hamcrest.Matcher;
+
+import io.restassured.http.Header;
+import io.vertx.core.json.JsonObject;
+
+public abstract class ProtectedEntityTestBase extends ApiTestBase {
+
+  static final String FULL_PROTECTED_USERS_UNIT_ID = "e68c18fc-833f-494e-9a0e-b236eb4b310b";
+  public static final List<String> PROTECTED_UNITS = Arrays.asList(FULL_PROTECTED_USERS_UNIT_ID);
+  static final String NOT_PROTECTED_UNIT_ID = "0e9525aa-d123-4e4d-9f7e-1b302a97eb90";
+  public static final List<String> NOT_PROTECTED_UNITS = Arrays.asList(NOT_PROTECTED_UNIT_ID, FULL_PROTECTED_USERS_UNIT_ID);
+  static final String NON_EXISTENT_UNIT_ID = "b548d790-07da-456f-b4ea-7a77c0e34a0f";
+  public static final List<String> NON_EXISTENT_UNITS = Arrays.asList(NON_EXISTENT_UNIT_ID, "0f2bb7a2-728f-4e07-9268-082577a7bedb");
+
+  private static final String USER_IS_NOT_MEMBER_OF_ORDERS_UNITS = "7007ed1b-85ab-46e8-9524-fada8521dfd5";
```

Comment:
Since this story is for an invoice, variable name should be something`USER_IS_NOT_MEMBER_OF_INVOICE_UNITS`
Same applies to variable below.

Reply:
Renamed to be more generic because acq units are going to be assigned to invoice and voucher records